packages/workspace/src/command-line/nx-commands.ts

```diff
@@ -286,6 +293,54 @@ function withAffectedOptions(yargs: yargs.Argv): yargs.Argv {
     });
 }
 
+function withRunManyOptions(yargs: yargs.Argv): yargs.Argv {
+  return yargs
+    .option('projects', {
+      describe: 'Projects to run',
+      type: 'string',
+      array: true,
+      requiresArg: true
+    })
+    .option('base', {
+      describe: 'Base of the current branch (usually master)',
+      type: 'string',
+      requiresArg: true
+    })
+    .option('head', {
+      describe: 'Latest commit of the current branch (usually HEAD)',
+      type: 'string',
+      requiresArg: true
+    })
+    .group(
+      ['base'],
+      'Run command using --base=[SHA1] (affected by the committed, uncommitted and untracked changes):'
+    )
+    .group(
+      ['base', 'head'],
+      'or using --base=[SHA1] --head=[SHA2] (affected by the committed changes):'
+    )
+    .options('runner', {
+      describe: 'This is the name of the tasks runner configured in nx.json',
+      type: 'string'
+    })
+    .options('configuration', {
+      describe:
+        'This is the configuration to use when performing tasks on projects',
+      type: 'string'
+    })
+    .options('only-failed', {
+      describe: 'Isolate projects which previously failed',
+      type: 'boolean',
+      default: false
+    })
+    .option('verbose', {
+      describe: 'Print additional error stack trace on failure'
+    })
+    .option('plain', {
```

Comment:
I don't think `affected:apps` and `affected:libs` will work with run-many. And this flag is only used by the two commands.